api/features/serializers.py

```diff
@@ -112,6 +112,9 @@ class ListCreateFeatureSerializer(DeleteBeforeUpdateWritableNestedModelSerialize
     )
     owners = UserListSerializer(many=True, read_only=True)
     group_owners = UserPermissionGroupSummarySerializer(many=True, read_only=True)
+
+    feature_state = serializers.SerializerMethodField()
```

Comment:
Let's be explicit with this attribute. It should be `environment_default_feature_state` or maybe just `environment_feature_state` is sufficient.

Reply:
Sounds good.